Livecheckables/scamper.rb

```diff
@@ -1,4 +1,4 @@
 class Scamper
-  livecheck :url   => "https://www.caida.org/tools/measurement/scamper/",
-            :regex => %r{href="code/scamper-cvs-([0-9\.]+)\.t}
+  livecheck :url   => "https://www.caida.org/tools/measurement/scamper/code/?C=M&O=D",
+            :regex => %r{href="scamper-cvs-([0-9\.]+)\.t}
```

Comment:
```
== /usr/local/Homebrew/Library/Taps/homebrew/homebrew-livecheck/Livecheckables/scamper.rb ==
141
C:  3: 23: Use // around regular expression.
```

This was failing `brew style`, since it was using the `%r{}` syntax without including a `/` in the regex. You should use `//` unless the regex includes a `/`, which is when you use the `%r{}` syntax. As mentioned before, running `brew style` on your changes before submitting your PR will help you avoid these issues.

```suggestion
            :regex => /href="scamper-cvs-([0-9\.]+)\.t/
```